start/Dockerfile.template

```
@@ -1,28 +1,25 @@
-FROM ubuntu:20.04
+FROM python:3.8
```

- does the katacode base have to be changed as well?
- should this be slim?

```suggestion
FROM python:3.8-slim
```